docs/en/Debugging-Guide.md

```diff
@@ -58,6 +59,48 @@ bucket, without the `s3://`, `s3a://`, or `s3n://` prefix.
 
 ## Usage FAQ
 
+#### Q: Why I see exceptions like "No FileSystem for scheme: alluxio"
+
+A: This error message is seen when your applications (e.g., MapReduce, Spark and etc) tried to access
+Alluxio as an HDFS-compatible data service, but the `alluxio://` scheme is not configured yet and thus not recognized by the
+application. Please make sure your HDFS configuration file `core-site.xml` (in your default hadoop
+installation or `spark/conf/` if you customize this file for Spark) has the following content:
+
+```xml
+<configuration>
+  <property>
+    <name>fs.alluxio.impl</name>
+    <value>alluxio.hadoop.FileSystem</value>
+  </property>
+</configuration>
+```
+
+#### Q: Why I see exceptions like "java.lang.RuntimeException: java.lang.ClassNotFoundException: Class alluxio.hadoop.FileSystem not found"
+
+A: This error message is seen when your applications (e.g., MapReduce, Spark and etc) tried to access
+Alluxio as an HDFS-compatible data service, the `alluxio://` scheme has been
+configured correctly but the Alluxio client jar is not found on the classpath of your application.
+
+- For MapReduce users, you can append the client jar to `$HADOOP_CLASSPATH`
+
+```bash
+$ export HADOOP_CLASSPATH=/<PATH_TO_ALLUXIO>/core/client/target/alluxio-core-client-{{site.ALLUXIO_RELEASED_VERSION}}-jar-with-dependencies.jar:${HADOOP_CLASSPATH}
+```
+
+- For Spark users, either expanding `$SPARK_CLASSPATH` by
```

Comment:
This sentence doesn't end, maybe change to `Spark users should extend '$SPARK_CLASSPATH'` with